test/integration/apimachinery/watch_restart_test.go

```diff
@@ -209,7 +209,6 @@ func TestWatchRestartsIfTimeoutNotReached(t *testing.T) {
 		for _, tmptc := range tt {
 			tc := tmptc // we need to copy it for parallel runs
 			t.Run(tc.name, func(t *testing.T) {
-				t.Parallel()
```

Comment:
Check the history, but I think this was added because these tests are long and we're making the package time really long

Reply:
fair enough, I'll check them separately